llama_stack/providers/registry/inference.py

```diff
@@ -52,6 +53,13 @@ def available_providers() -> List[ProviderSpec]:
             module="llama_stack.providers.inline.inference.vllm",
             config_class="llama_stack.providers.inline.inference.vllm.VLLMConfig",
         ),
+        InlineProviderSpec(
+            api=Api.inference,
+            provider_type="inline::sentence-transformers",
```

Comment:
Should we provide `inline::sentence-transformers` as a default inference provider for backward compatibility?

Reply:
@yanxi0830 `inline::meta-reference` also implements this functionality and is how we get backwards compatibility. But are you saying this for distributions which don't use `inline::meta-reference`?

Reply:
E.g. how should `together` + `faiss` work together?

Reply:
actually, faiss can work with together's embedding models.

Reply:
The explicit sentence transformer provider is only for cases like TGI where there is no way to have a hosted embedding model .